test/integration/openid4ci_test.go

```diff
@@ -109,11 +114,36 @@ func TestOpenID4CIFullFlow(t *testing.T) {
 		require.NoError(t, err)
 		require.Contains(t, vc.VC.Issuer.ID, tc.issuerDIDMethod)
 
-		subID, err := verifiable.SubjectID(vc.VC.Subject)
-		require.Contains(t, subID, didID)
+		if !tc.isSD {
+			subID, err := verifiable.SubjectID(vc.VC.Subject)
+			require.NoError(t, err)
+			require.Contains(t, subID, didID)
+			checkActivityLogAfterOpenID4CIFlow(t, activityLogger, tc.issuerProfileID, subID)
+		} else {
+			subID := getDisclosureHelper(t, serializedVC, "id")
+			require.Contains(t, subID, didID)
```

Comment:
You can remove the else and move the body of the else outside

Reply:
getDisclosureHelper will fail for regular VC

Reply:
My bad - when I wrote my comment, I thought your `if` block ended in a return, but I see now that it doesn't. What I was thinking of was this:
```
if !tc.isSelectiveDisclosure {
	subID, err := verifiable.SubjectID(vc.VC.Subject)
	require.NoError(t, err)
	require.Contains(t, subID, didID)
	checkActivityLogAfterOpenID4CIFlow(t, activityLogger, tc.issuerProfileID, subID)
	return
}

subID := getDisclosure(t, serializedVC, "id")
			require.Contains(t, subID, didID)
```